### PR TITLE
Move ruby_install to languages cookbook.

### PR DESCRIPTION
**Pull request by tylercloke**
Ping @chef-cookbooks/delivery-migration-team 

Relates to https://github.com/chef-cookbooks/languages/pull/4

TODO
- [x] Remove chef-install installation since `chef_install` does that for you now? Main difference between this and `languages` implementation is `-j #{node.builders}`. https://github.com/chef-cookbooks/omnibus/blob/master/recipes/_ruby.rb#L38-L46

### CODE REVIEW

**Comment by chef-delivery** [comment]
This PR has passed 'Verify' and is ready for review and approval!
  Use: '@delivery approve' when code review is complete.

**Comment by schisamo** [comment]
We should be able to validate this by running:

```
kitchen verify default-ubuntu-1404
```

**Comment by schisamo** [comment]
I think we need to nuke:
https://github.com/chef-cookbooks/omnibus/blob/aedb966e1507c048572f3ff1ee642d93d2c0de14/recipes/_ruby.rb#L38-L46

File: Berksfile [hunk 1]
```diff
@@ -6,6 +6,8 @@ source 'https://supermarket.chef.io'
 
 metadata
 
+cookbook 'languages', git: 'https://github.com/chef-cookbooks/languages.git', branch: 'ruby_install'
```
Comment (tylercloke) [inline Berksfile]: Probably not the best way to do this?

Comment (schisamo) [inline Berksfile]: I'll get `languages` publishing to supermarket.chef.io. That will ensure we don't have to cargo cult `Berksfile` entries all over the place (including Delivery build cookbooks).

**Comment by chef-delivery** [comment]
This PR has passed 'Verify' and is ready for review and approval!
  Use: '@delivery approve' when code review is complete.

**Comment by ksubrama** [comment]
:+1:

**Comment by chef-delivery** [comment]
This PR has passed 'Verify' and is ready for review and approval!
  Use: '@delivery approve' when code review is complete.

**Comment by chef-delivery** [comment]
This PR has passed 'Verify' and is ready for review and approval!
  Use: '@delivery approve' when code review is complete.

**Comment by chef-delivery** [comment]
This PR has passed 'Verify' and is ready for review and approval!
  Use: '@delivery approve' when code review is complete.

**Comment by schisamo** [comment]
@delivery approve

**Comment by chef-delivery** [comment]
Change: ca3d01fb-0bea-489c-a0de-ab3d3247c9b8 approved by: @schisamo

**Comment by chef-delivery** [comment]
This PR has passed 'Acceptance' and is ready to be delivered!
  Use: '@delivery deliver' when validated in acceptance.

**Comment by schisamo** [comment]
@delivery deliver

**Comment by chef-delivery** [comment]
Change: ca3d01fb-0bea-489c-a0de-ab3d3247c9b8 delivered by: @schisamo

**Pull request merged**